e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -189,12 +188,80 @@ public async Task RegistryManager_AddDeviceWithProxy()
             await registryManager.AddDeviceAsync(device).ConfigureAwait(false);
         }
 
+        [LoggedTestMethod]
+        public async Task RegistryManager_ConfigurationOperations_Work()
+        {
+            // arrange
+
+            bool configCreated = false;
+            string configurationId = (_idPrefix + Guid.NewGuid()).ToLower(); // Configuration Id characters must be all lower-case.
```

Comment:
is this requirement enforced by the service? should we document that in the `Configuration` constructor and the Id property?

Reply:
Yes. Added documentation to the ctor. As it can't be set on the Id property, I didn't carry the documentation over to the property.